clients/src/main/java/org/apache/kafka/common/Uuid.java

```diff
@@ -31,7 +31,7 @@
  * This is the same type of UUID as the ones generated by java.util.UUID. The toString() method prints
  * using the base64 string encoding. Likewise, the fromString method expects a base64 string encoding.
  */
-public class Uuid implements Comparable<Uuid> {
+public class Uuid implements Comparable<Uuid>, java.io.Serializable {
```

Comment:
I don't feel comfortable with this change. `Uuid` is part of our public API so we would have to keep it forever.